### PR TITLE
borderStyle:"dashed"

### PR DESCRIPTION
**Pull request by rkyle35242**
Inclusion of the {borderStyle:"dashed"} style.

### CODE REVIEW

**Comment by adkenyon** [comment]
Yes, let's add all the variants!

**Comment by CoryWritesCode** [comment]
@euroclydon37 `br` defaults to solid so that might not need support.

**Comment by euroclydon37** [comment]
@CoryWritesCode We still need to support it because styled components can be composed together and you might need to override a previous border styling.
```js
const borderedContainer = styled(View)`br3 w5, h3 bg-blue white b--gray bs-dash`

// want everything but the dashed border style
const solidBorderedContainer = styled(borderedContainer)`bs-solid`
```

File: src/styles/borders.js [hunk 1]
```diff
@@ -24,5 +24,8 @@ export default {
   bl: { borderLeftWidth: 1 },
   br: { borderRightWidth: 1 },
   bt: { borderTopWidth: 1 },
-  bb: { borderBottomWidth: 1 }
+  bb: { borderBottomWidth: 1 },
+  bs_dashed: { borderStyle: 'dashed' },
+  bs_dotted: { borderStyle: 'dotted' },
+  bs_solid: { borderStyle: 'solid' }
```
Comment (euroclydon37) [inline src/styles/borders.js]: Perfect. Use hyphens instead of underscores, though, for consistency.

**Comment by adkenyon** [comment]
One comment, I changed all the sizing when recently. We're going to need to do an extensive find/replace to handle it. And of course, re learn the sizing numbers....

**Comment by rkyle35242** [comment]
@adkenyon I didn't touch any sizing here. Are you wanting me to make those changes on this PR?

**Comment by adkenyon** [comment]
What I mean is that, on my previous PR on the repo I did update sizing. So if we want to publish this and use it in Nebula we will need to update the sizing in Nebula

**Comment by rkyle35242** [comment]
Oh boy

**Pull request merged**